app/routes/sign.py

```diff
@@ -1,25 +1,23 @@
-from flask import request,jsonify,Blueprint
-from app.util import email_check,user_check
-from app.dob import insert_user
-second=Blueprint("/second",__name__)
+from flask import request , jsonify , Blueprint
+from app.util import  email_check , user_check
+from app.dob import  insert_user
 
-@second.route('/signup', methods =['POST'])
+sign=Blueprint("/sign",__name__)
```

Comment:
We can keep it as auth, Since signin, signup Both comes under authentication and authorization functionalities.